src/app/educate-teacher/my-classes/components/resources/resources.component.html

```diff
@@ -19,18 +19,32 @@
     </div>
   </div>
   <div class="resources-body" appCardBodyStyle="23px 0px">
```

Comment:
`appCardBodyStyle` doesn't require "23px 0px".
We are not providing "23px 0px" anywhere else.

Reply:
i have difine 23px because, i am using padding 10px in items(resources) so it wil be 23px + 10px = 33 px , which we use everywhere